docker/nginx/default.conf

```diff
@@ -1,13 +1,26 @@
 server {
-  listen 80 default_server;
+  listen 80 default_server reuseport;
 
   keepalive_timeout 70;
 
   index index.php;
 
-  location / {
-    root /usr/share/nginx/html;
-    try_files $uri /index.php$is_args$args;
+  root /usr/share/nginx/html;
+  try_files $uri /index.php$is_args$args;
+
+  # security headers
+  add_header X-XSS-Protection          "1; mode=block" always;
+  add_header X-Content-Type-Options    "nosniff" always;
+  add_header Referrer-Policy           "no-referrer-when-downgrade" always;
+  add_header Content-Security-Policy   "default-src 'self' http: https: ws: wss: data: blob: 'unsafe-inline'; frame-ancestors 'self';" always;
+  add_header Permissions-Policy        "interest-cohort=()" always;
+  add_header Strict-Transport-Security "max-age=31536000; includeSubDomains; preload" always;
+
+  location ~* \.(css|png|ico|webmanifest|eot|svg|ttf|woff|woff2|txt)$ {
+    try_files  $uri /index.php$is_args$args;
+    access_log off;
+    expires    max;
+    add_header Cache-Control "public";
```

Comment:
I know it was generated automatically, but I wonder if we have any use for this section.

Reply:
This section is used to make nginx cache static assets :)

It sends the request to `/index.php$is_args$args` for badges (they end in `.svg`).